sympy/matrices/matrices.py

```diff
@@ -2887,10 +2887,19 @@ def gauss_jordan_solve(self, b, freevar=False):
         Matrix([
         [tau0],
         [tau1]])
+        >>> taus_zeroes = { tau:0 for tau in params }
+        >>> sol_unique = sol.xreplace(taus_zeroes)
+        >>> sol_unique
+         Matrix([
+        [2],
+        [0],
+        [5],
+        [0]])       
```

Comment:
```
________________________________________________________________________________
____________ sympy/utilities/tests/test_code_quality.py:test_files _____________
Traceback (most recent call last):
  File "/home/travis/build/sympy/sympy/sympy/utilities/runtests.py", line 1306, in _timeout
    function()
  File "/home/travis/build/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 242, in test_files
    check_directory_tree(SYMPY_PATH, test, exclude)
  File "/home/travis/build/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 109, in check_directory_tree
    check_files(glob(join(root, pattern)), file_check, exclusions)
  File "/home/travis/build/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 125, in check_files
    file_check(fname)
  File "/home/travis/build/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 146, in test
    test_this_file(fname, test_file)
  File "/home/travis/build/sympy/sympy/sympy/utilities/tests/test_code_quality.py", line 172, in test_this_file
    assert False, message_space % (fname, idx + 1)
AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/matrices/matrices.py, line 2897.
```

Reply:
It may be the next empty line (the fact that there are two empty lines). Not clear from the diff where they may be.

Reply:
Ah, I noted that you had already pushed a new version so I cancelled the old tests so that the new one will start earlier. (This was from an earlier version.)

Reply:
Thanks, you are right, for some reason there were some trailing spaces left out. Should be fixed now.